serverlessworkflow/modules/ROOT/pages/cloud/operator/external_resources_configuration_for_workflow_CR_devprofile.adoc

```diff
@@ -0,0 +1,218 @@
+= Support external resources configuration in development profile
+:compat-mode!:
+// Metadata:
+:description: Support external resources configuration in development mode
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, config, openshift, containers, development
+
+This document describes how to use external resources configuration for Workflow Custom Resource in development mode with the {operator_name}.
+
+== Configure external files support
```

Comment:
Can you please expand this guide? I understand that we should include the following:

- Explain why this use case is important. It's important because, without it, users can't deploy workflows that depend on external resources such as OpenAPI, AsyncAPI, Camel Routes, or any other resource like a data input schema. See these guides about service orchestration: https://kiegroup.github.io/kogito-docs/serverlessworkflow/1.38.0.Final/service-orchestration/orchestration-of-openapi-based-services.html
- The example is nice, but we should go through each step with more details

Have you run this example locally?